python/python/lancedb/remote/table.py

```diff
@@ -114,7 +114,7 @@ def create_scalar_index(
         index_type: Literal["BTREE", "BITMAP", "LABEL_LIST", "scalar"] = "scalar",
         *,
         replace: bool = False,
-        wait_timeout: timedelta = None,
```

Comment:
could we do the same for create_fts_index as well?

Reply:
Oh yeah, good catch! fixed